app/assets/javascripts/application.js

```diff
@@ -13,3 +13,5 @@
 //= require jquery
 //= require jquery_ujs
 //= require_tree .
+
+window.GAAP.analytics.init();
```

Comment:
Do we need to require the `gaap-analytics.js` file in here too?

Reply:
`//= require_tree` brings in everything from the application directory

Reply:
You can remove `//= require_tree .` and specify the files individually when you want to control the load order.